src/snowflake/snowpark/modin/plugin/extensions/groupby_overrides.py

```diff
@@ -508,9 +508,28 @@ def diff(self):
     def ffill(self, limit=None):
         ErrorMessage.method_not_implemented_error(name="ffill", class_="GroupBy")
 
-    def fillna(self, *args, **kwargs):
+    def fillna(
+        self,
+        value: Any = None,
+        method: Optional[FillnaOptions] = None,
+        axis: Optional[Axis] = None,
+        inplace: Optional[bool] = False,
+        limit: Optional[int] = None,
+        downcast: Optional[dict] = None,
+    ):
         # TODO: SNOW-1063349: Modin upgrade - modin.pandas.groupby.DataFrameGroupBy functions
-        ErrorMessage.method_not_implemented_error(name="fillna", class_="GroupBy")
```

Comment:
Do we know if `SeriesGroupBy.fillna` will be supported or not? If not, then we should still raise an error.

Reply:
I added support for Series.groupby.fillna as get it from same implementation.